### DISCUSSION TITLE
SumatraPDF Preview Error (*.cbt)

### DISCUSSION THREAD

**Discussion by rontarrant** [sumatrapdfreader/sumatrapdf#3620] [Q&A] [answered]
I've created a bunch of .CBZ files using a Python script, but they won't preview in Windows Explorer's preview pane.
I used Python's ZipFile library to combine a series of jPeg files into a .zip. I then renamed the extension to .cbz and .cbz files all show their covers as icons on Windows Explorer's left-hand pane. But clicking on any one file to preview in the right-hand pane results in this error:

_This file can't be previewed because of an error in the SumatraPDF Preview (*.cbt)._

Can anybody shed light on this?

PS: I've tried this with .cbr and .cbz files I've picked up elsewhere, too. Same result.

**Comment by GitHubRulesOK** [comment, marked as answer]
cbX (cbz cbr cbt cb7) are archives with images 
the main SumatraPDF previewer normally only previews PDF
not any of the other supported previews,
I think at some time the ePub preview engine was active but have not seen that recently

To activate the different previewers they need to be activated during compile
 but there are still a few oddities last time (a few years ago) I used a full set https://github.com/sumatrapdfreader/sumatrapdf/issues/1676

**Reply by rontarrant** [reply]
Okay, thanks.
